company/okrs/2020_q1.md

```diff
@@ -108,10 +108,10 @@
    1. At least four customers are running AND/OR/NOT queries and metrics to validate this KR are tracked in [pings](https://docs.sourcegraph.com/admin/pings).
 
 1. **Transparent and scalable code-host permissions syncing**
-   1. 95th percentile latency of authorization on the read path (i.e. `authzFilter`) is below 10ms. => Prometheus metrics in place, need to send this metric via ping.
+   1. 95th percentile latency of authorization on the read path (i.e. `authzFilter`) is below 10ms. => Prometheus metrics in place, not able to add to ping because of technical diffcuities. Instead, will add alerts to Grafana panels.
```

Comment:
```suggestion
   1. 95th percentile latency of authorization on the read path (i.e. `authzFilter`) is below 10ms. => Prometheus metrics in place, not able to add to ping because of technical difficulties. Instead, will add alerts to Grafana.
```